comms/examples/stress_test.rs

```diff
@@ -56,6 +56,13 @@ async fn run() -> Result<(), Error> {
 
     let is_tcp = remove_arg(&mut args, "--tcp").is_some();
 
+    let mut rate_limit = 100_000;
+    println!("Default rate limit is: {}/s", rate_limit);
+    if let Some(pos) = remove_arg(&mut args, "--rate-limit") {
+        rate_limit = args.remove(pos).parse().expect("Unable to parse rate limit");
+        println!("Rate limit reset to:   {}/s", rate_limit);
+    }
+
```

Comment:
Why not just output what the rate limit is set to?

```suggestion
    let mut rate_limit = 100_000;
    if let Some(pos) = remove_arg(&mut args, "--rate-limit") {
        rate_limit = args.remove(pos).parse().expect("Unable to parse rate limit");
    }
        println!("Rate limit set to:   {}/s", rate_limit);
```